Submissions/Ujjwal0501/Assignment-1/answers.txt

```diff
@@ -1 +1,110 @@
-Write your answers in a text file.
+Assignment 1
```

Comment:
Again you are not supposed to change contents of other's files.

- Restore the original content of this file.